pkg/database/realm.go

```diff
@@ -1384,46 +1433,55 @@ func (r *Realm) CreateSigningKeyVersion(ctx context.Context, db *Database) (stri
 // and the key manager. ID is the primary key ID from the database. If the id
 // does not exist, it does nothing.
 func (r *Realm) DestroySigningKeyVersion(ctx context.Context, db *Database, id interface{}) error {
+	signingKey := &SigningKey{}
+	return r.destroyManagedSigningKey(ctx, db, id, "signing_keys", "certificate", signingKey)
+}
+
+func (r *Realm) DestroySMSSigningKeyVersion(ctx context.Context, db *Database, id interface{}) error {
+	signingKey := &SMSSigningKey{}
+	return r.destroyManagedSigningKey(ctx, db, id, "sms_signing_keys", "SMS", signingKey)
+}
+
+func (r *Realm) destroyManagedSigningKey(ctx context.Context, db *Database, id interface{}, table, purpose string, signingKey ManagedKey) error {
```

Comment:
I think you can call Purpose() instead of passing it in

Reply:
oops - missed the destroy method